modules/administration-guide/pages/installing-che-in-a-restricted-environment.adoc

```diff
@@ -1,26 +1,35 @@
 :_content-type: PROCEDURE
-:navtitle: Installing {prod-short} in restricted environment
+:navtitle: Installing {prod-short} in a restricted environment
 :keywords: administration guide, installing-che-in-a-restricted-environment, installing-che-in-restricted-environment
-:page-aliases: installation-guide:installing-che-in-restricted-environment, installation-guide:installing-che-in-a-restricted-environment
+:page-aliases: installation-guide:installing-che-in-restricted-environment, installation-guide:installing-che-in-a-restricted-environment, preparing-a-restricted-environment, configuring-che-to-run-in-a-restricted-environment
 
-:parent-context-of-installing-che-in-a-restricted-environment: {context}
 
-[id="installing-{prod-id-short}-in-a-restricted-environment_{context}"]
-= Installing {prod-short} in a restricted environment
 
-:context: installing-{prod-id-short}-in-a-restricted-environment
+[id="installing-{prod-id-short}-in-a-restricted-environment-on-openshift_{context}"]
+= Installing {prod-short} in a restricted environment on OpenShift
 
-By default, {prod-short} workspaces reference various external resources, including container images available in public registries or sample projects hosted on public resources. This section describes how to deploy {prod-short} in a restricted environment where these external resources are not available. This method is also called _offline mode_ or _air-gap scenario_.
+On an OpenShift cluster operating in a restricted network, public resources are not available.
 
-.Prerequisites
+Deploying {prod-short} and running workspaces requires public resources:
```

Comment:
```suggestion
Deploying {prod-short} and running workspaces requires the following public resources:
```
- Add `following` to indicate that the bulleted list includes all of the required resources